src/udp_wrap.cc

```diff
@@ -139,7 +139,7 @@ void UDPWrap::New(const FunctionCallbackInfo<Value>& args) {
 
 
 void UDPWrap::GetFD(Local<String>, const PropertyCallbackInfo<Value>& args) {
-  int fd = -1;
+  int fd = UV_EINVAL;
```

Comment:
@bnoordhuis You had me originally change this to `UV_EINVAL`, but almost everywhere else it's `UV_EBADF`. Should this stay as is?